Document typed cards (#5548)
mintlify/docs: components/cards.mdx

</Card>
```

### Typed cards

Add a `type` property to style a card with the same theme as a [callout](/components/callouts). Typed cards apply a colored background, border, and default icon that match the selected type, so you can use them to draw attention to grouped content like notes, warnings, or success states.

> Supported types: `info`, `warning`, `note`, `tip`, `check`, and `danger`. The matching icon is used automatically, but you can override it by passing your own `icon`.

Reviewer: ```suggestion
Supported types: `info`, `warning`, `note`, `tip`, `check`, and `danger`. Optionally, override the default icon with the `icon` property.
```
